spyder/widgets/status.py

```diff
@@ -378,7 +379,6 @@ def _get_env_info(self, path):
             version = get_interpreter_info(path)
             self.path_to_env[path] = name
             self.envs[name] = (path, version)
-        __, version = self.envs[name]
```

Comment:
> I don't understand how this could have happened. Could you describe how do you think the env was removed?

I put a `print(self.envs)` here (outside the try/except).
Here is the result:
```
{'custom': ('/usr/bin/python3', 'Python 3.8.5')}
{}
{}
```